pkg/reconciler/kubernetes/tektonconfig/controller.go

```diff
@@ -28,7 +28,11 @@ import (
 
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
+	tektonAddoninformer "github.com/tektoncd/operator/pkg/client/injection/informers/operator/v1alpha1/tektonaddon"
```

Comment:
this is openshift specific so we can probably move this in https://github.com/tektoncd/operator/blob/7490fc2095c004f17c1946175682cb938b7efb18/pkg/reconciler/openshift/tektonconfig/controller.go#L33